pkgs/ok_http/lib/src/ok_http_client.dart

```diff
@@ -21,6 +21,48 @@ import 'package:jni/jni.dart';
 
 import 'jni/bindings.dart' as bindings;
 
+/// Configurations for the [OkHttpClient].
+class OkHttpClientConfiguration {
+  /// The maximum duration to wait for a call to complete.
+  ///
+  /// If a call does not finish within the specified time, it will throw a
+  /// [ClientException] with the message "java.io.InterruptedIOException...".
+  /// Default is set to [Duration.zero], which indicates no timeout.
+  ///
+  /// See [OkHttpClient.Builder.callTimeout](https://square.github.io/okhttp/5.x/okhttp/okhttp3/-ok-http-client/-builder/call-timeout.html).
+  final Duration callTimeout;
+
+  /// The maximum duration to wait while connecting a TCP Socket to the target
+  /// host.
+  ///
+  /// Default is set to `10000` milliseconds.
```

Comment:
I would remove the docs for the default because they will become out-of-sync with the constructor.